index.bs

```diff
@@ -1458,11 +1458,15 @@ as appropriate.
 
 
 
-<h3 id="security">Security</h3>
+<h3 id="extension-security-and-privacy">Security and Privacy</h3>
 
-All interfaces defined by extension specifications
-should only be available within a [=secure context=].
+Extension specifications are expected to:
 
+- conform with the generic [[#mitigation-strategies|mitigation strategies]],
+- consider [[#mitigation-strategies-case-by-case|mitigation strategies applied
+  on a case by case basis]],
+- be evaluated against the Self-Review Questionnaire on Security and Privacy
+  [[SECURITY-PRIVACY-QUESTIONNAIRE]].
```

Comment:
maybe specifically call-out the same-origin policy checkpoint? otherwise lgtm

Reply:
Thanks for the feedback. Added explicit checkpoint in https://github.com/w3c/sensors/pull/275/commits/4f5f2619dd1d5c8c63b38b67593ce1004ee66cdc